README.md

```diff
@@ -141,6 +141,7 @@ Detailed examples can be found in the [examples](/examples) folder with the foll
 - [Kafka Component, HTTP Component, HTTP Authentication, Kafka Tracing](/examples/second/main.go)
 - [Kafka Component, AMQP Tracing](/examples/third/main.go)
 - [AMQP Component](/examples/fourth/main.go)
```

Comment:
The fourth example should send a message to SNS, which is coupled with SQS, and the fifth should process it.